marklogic-data-hub-central/ui/e2e/cypress/e2e/explore/exploreXMLValidations.cy.tsx

```diff
@@ -247,8 +251,11 @@ describe("xml scenario for snippet view on browse documents page", () => {
     browsePage.getSearchText().clear();
   });
 
-  it.skip("Verify metadata view of the document", () => {
+  it("Verify metadata view of the document", () => {
+    browsePage.clickTableView();
+    cy.waitForAsyncRequest();
     browsePage.getSearchText().clear();
+    browsePage.getApplyFacetsButton().click();
```

Comment:
Why are we appling faces here? Read my suggestion of removing all this clear and re-searching for "Bowman". Looks like it is not needed.